packages/tasks/src/local-apps.ts

```diff
@@ -105,7 +105,13 @@ export const LOCAL_APPS = {
 		 * random function, will need to refine the actual conditions:
 		 */
 		displayOnModelPage: (model) => model.library_name === "diffusers" && model.pipeline_tag === "text-to-image",
```

Comment:
```suggestion
		displayOnModelPage: (model) => model.library_name === "diffusers" && (model.pipeline_tag === "text-to-image" || model.tags.includes("lora")),
```
As per https://huggingface.slack.com/archives/C03RVEGLNBW/p1715780731909589?thread_ts=1713958091.775499&cid=C03RVEGLNBW